### ISSUE TITLE
vocal accents

### ISSUE THREAD

**Issue by wfpaisa** [balderdashy/waterline#1242]
In spanish use aeiou and áéíóú 
How I can do searches with accents?

Thank you

**Comment by devinivy** [comment]
This should be adapter- and even database-dependent.  Which adapter do you use?  If it's an adapter for a sequel database, look into adjusting you database or table's collation.  As far as I know, waterline itself is agnostic to searching with diacritics.

**Comment by wfpaisa** [comment]
I use sails-mongo

**Comment by sailsbot** [comment]
Thanks for posting, @wfpaisa.  I'm a repo bot-- nice to meet you!

It has been 30 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
